app/client/keybase/debug/keystore.go

```diff
@@ -134,7 +133,7 @@ func fetchValidatorPrivateKeysFromFile() (map[string]string, error) {
 	_, current, _, _ := r.Caller(0)
 	//nolint:gocritic // Use path to find private-keys yaml file from being called in any location in the repo
 	yamlFile := filepath.Join(current, privateKeysYamlFile)
-	if exists, err := shared.FileExists(yamlFile); !exists || err != nil {
+	if exists, err := converters.FileExists(yamlFile); !exists || err != nil {
```

Comment:
Just FYI: Going to be cleaning this up in my upcoming PR (making a couple changes to the util).

No action.